staff.html

```diff
@@ -66,7 +66,7 @@
                           <i class="fa fa-twitter"></i>
                       </span>
               </a>
-              <a class="nav-item" href="https://www.discord.gg/ssl">
+              <a class="nav-item" href="https://sslcommunity.io/sslcommunity">
```

Comment:
Is this link even valid? I don't see any change here that would cause that to redirect...

Reply:
Sorry mistake on my part, accidently wrote the wrong thing. Will fix now